src/nowsecure-sarif.ts

```diff
@@ -14,10 +14,11 @@ import type {
 } from "sarif";
 import * as core from "@actions/core";
 import crypto from "crypto";
-import type { Finding, PullReportResponse } from "./nowsecure-types";
+import type { Finding, PullReportResponse } from "./types/platform";
 import { ripGrep as rg, RipGrepError } from "ripgrep-js";
 import { getConfig } from "./utils/config-check";
 
+const DEFAULT_LAB_URL = "https://lab.nowsecure.com";
```

Comment:
in the future wondering if we should centralize these URLs somewhere else or into the client

Reply:
Yeah this is was tricky because it's not a URL used by the client, it is only used for generating informational URLs so I put it in the SARIF conversion for that reason. It can also be overridden in the Action parameters which made it more difficult to decide.